src/layout/barGrid.js

```diff
@@ -478,7 +478,8 @@ export function layout(seriesType, ecModel) {
                 x: x,
                 y: y,
                 width: width,
-                height: height
+                height: height,
+                rb: seriesModel.get('backgroundStyle.barBorderRadius')
             });
```

Comment:
`radius` is not a layout related property. So I think it's better to set the property here.

https://github.com/apache/incubator-echarts/blob/b4b979a2fb9ad97493e7efb7c4d64055325ad214/src/chart/bar/BarView.js#L146

```js
bgEl.setShape('r', backgroundModel.get('barBorderRadius'));
```